openlibrary/templates/books/breadcrumb_select.html

```diff
@@ -1,17 +1,39 @@
-$def with (key=None, selected="", options=None)
+$def with (mb, selected="books")
+
+    $ url_prefix = "/people/" + mb.username
+    $ readlog_url = url_prefix + "/books/%s/"
+    $ options = [(_("Books"), url_prefix + "/books")]
+    $if mb.is_my_page or mb.is_public:
+      $ options += [
+      $   (_("Want to Read (%(count)d)", count=mb.counts['want-to-read']), readlog_url % "want-to-read"),
+      $   (_("Currently Reading (%(count)d)", count=mb.counts['currently-reading']), readlog_url % "currently-reading"),
+      $   (_("Already Read (%(count)d)", count=mb.counts['already-read']), readlog_url % "already-read")
+      $ ]
+    $ options += [(_("Lists (%(count)d)", count=len(mb.lists)) , url_prefix + "/lists")]
+    $if mb.is_my_page:
+      $ options += [
+      $   (_("Loans"), "/account/loans"),
+      $   (_("Notes"), url_prefix + "/books/notes"),
+      $   (_("Notes"), url_prefix + "/books/notes"),
+      $   (_("Reviews"), url_prefix + "/books/observations"),
+      $   (_("Imports and Exports"), "/account/import")
+      $ ]
+    $if mb.sponsorships:
+      $ options += [ (_("Sponsorships"), readlog_url % "sponsorships") ]
+    $ prefix = _("My") if mb.is_my_page else "%s's" % mb.username
```

Comment:
This is un-i18nable, and I don't think serves a strong purpose. There's plenty of context to denote that this is "my" stuff.